src/theory/bags/infer_info.h

```diff
@@ -81,8 +81,8 @@ class InferInfo : public TheoryInference
   bool process(TheoryInferenceManager* im, bool asLemma) override;
   /** The inference identifier */
   Inference d_id;
-  /** The conclusion */
-  Node d_conclusion;
+  /** The conclusions */
+  std::vector<Node> d_conclusions;
```

Comment:
I don't think we should make this change.  If necessary, I would make multiple conclusions into an AND node.  This is what strings does.

This is important since this class will eventually be merged with a standard InferInfo class, which will not have a vector of conclusions.

Reply:
I suggest changing the standard InferInfo to have a vector of conclusions which is more flexible. 
In some situations, I don't know in advance how many inferences I need. To use AND I have to start with a default conclusion (TRUE) in this case. Then I need to add ugly code to check for this case. 
Empty conclusions is nicely handled in the process function